lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -833,6 +845,107 @@ def test_riscv64_regs_gpr_only(self):
             substrs=["registers were unavailable"],
         )
 
+    @skipIfLLVMTargetMissing("LoongArch")
```

Comment:
Is LoongArch one LLVM target or do you need to specifically have LoongArch64 for this?